scripts/data_analysis/README.md

```diff
@@ -34,8 +35,33 @@ To run this script *from within the tooling folder*, based on your python distri
 ```shell
 python3 retrieve_schema_fields.py
 ```
-Outputs:
-A csv file in the [outputs](outputs) directory containing
+>Outputs:
+>A csv file in the [outputs](outputs) directory containing schemas and fields within them
```

Comment:
```suggestion
>Outputs:
> - A csv file in the [outputs](outputs) directory containing schemas and fields within them
```

I know it's only one output for this one but to make it consistent with outputs of other script and visually imply that there could be multiple outputs?

Reply:
Agreed.